.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+on: push
+jobs:
+  Build_Project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: pwsh ./get-dependencies.ps1
+      - run: pwsh ./build.ps1
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.sha }}
+          path: |
+            *.class
+            *.jar
+  Run_Basic_IO_Tests:
+    needs: Build_Project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v2
+        with: 
+          name: ${{ github.sha }}
+          path: '.'  
+      - run: pwsh ./run.ps1 < test/test1.txt
+      - run: pwsh ./run.ps1 < test/test2.txt
+      - run: pwsh ./run.ps1 < test/test3.txt
+      - run: pwsh ./run.ps1 < test/test4.txt
+      - run: pwsh ./run.ps1 < test/test5.txt
+      - run: pwsh ./run.ps1 < test/test6.txt
+      - run: '! pwsh ./run.ps1 < test/test7.txt'
+      - run: pwsh ./run.ps1 test/test8.json test/test9.json
+      - run: '! pwsh ./run.ps1 test/test8.json'
+      - run: pwsh ./run.ps1 test/test9.json test/test9.json
+      - run: pwsh ./run.ps1 test/test9.json test/test10.json
```

Comment:
Ez oke, azt nem latom, hol van tesztelve, hogy a vart kimenetet kapjuk.

Reply:
Szia @hegyhati ! Mostmár megvan a diff része is, kérlek ellenőrizd.